includes/blocks/course-theme/class-notices.php

```diff
@@ -49,7 +49,9 @@ public function __construct() {
 	 * @return string The block HTML.
 	 */
 	public function render( array $attributes = [] ) : string {
-		$notices_html = Sensei_Context_Notices::instance( 'course_theme_lesson_quiz' )->get_notices_html( 'course-theme/lesson-quiz-notice.php' )
+		$notices_html = Sensei_Context_Notices::instance( 'course_theme_lesson_regular' )->get_notices_html( 'course-theme/lesson-quiz-notice.php' )
+			. Sensei_Context_Notices::instance( 'course_theme_lesson_quiz' )->get_notices_html( 'course-theme/lesson-quiz-notice.php' )
+			. Sensei_Context_Notices::instance( 'course_theme_locked_lesson' )->get_notices_html( 'course-theme/locked-lesson-notice.php' )
```

Comment:
Are we adding `course_theme_locked_lesson` twice?

Reply:
Oops. I didn't event notice that part. Nice catch! Updated. 55d2f5a9c1caa44cef2eb5f100b36771ef5aced3